app/src/prodRelease/play/release-notes/en-US/default.txt

```diff
@@ -1 +0,0 @@
-Include release notes with alpha releases
```

Comment:
I'm not sure what removing this file is doing, or whether it's related to this PR?